### PR TITLE
banner item and carousel added

### PR DESCRIPTION
**Pull request by FaaiqKhan**

### CODE REVIEW

File: squatwolf-mobile-app/src/assets/icons/LeftArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Svg, { Path } from "react-native-svg"
+
+interface ILeftArrow {
+  isDisable: boolean
+}
+
+export const LeftArrow = ({ isDisable }: ILeftArrow) => {
+  const fillColor = isDisable ? "#121212" : "#B4B4B4"
```
[the remark this answers is not in the document]
Comment (hamza-akram) [reply squatwolf-mobile-app/src/assets/icons/LeftArrow.tsx:8]: Add empty line between const and return for better readability.

File: squatwolf-mobile-app/src/assets/icons/RightArrow.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Svg, { Path } from "react-native-svg"
+
+interface IRightArrow {
+  isDisable: boolean
+}
+
+export const RightArrow = ({ isDisable }: IRightArrow) => {
+  const fillColor = isDisable ? "#121212" : "#B4B4B4"
```
Comment (hamza-akram) [inline squatwolf-mobile-app/src/assets/icons/RightArrow.tsx:8]: Same

File: squatwolf-mobile-app/src/components/BannerCarousel.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import { BannerItem, IBannerDimension } from "./BannerItem"
+import { IconButton } from "react-native-paper"
+import { LeftArrow } from "../assets/icons/LeftArrow"
+import { RightArrow } from "../assets/icons/RightArrow"
+import { createStyleSheet, useStyles } from "react-native-unistyles"
+
+export const BannerCarousel = ({ data }: IHerosSection) => {
+  const { styles } = useStyles(stylesheet)
+
+  const bannerItemDelta = 50
```
Comment (hamza-akram) [inline squatwolf-mobile-app/src/components/BannerCarousel.tsx:13]: Can we move this into constant file

File: squatwolf-mobile-app/src/components/BannerItem.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+  const sectionData = data as ISection
+
+  return (
+    <View>
+      <Image
+        style={{
+          width: dimension?.width ?? windowWidth,
+          height: dimension?.height ?? windowWidth * 1.2,
+          backgroundColor: "pink"
```
Comment (hamza-akram) [inline squatwolf-mobile-app/src/components/BannerItem.tsx:26]: Can we add color code constant here rather than "pink"?

**Review by hamza-akram — CHANGES_REQUESTED** [review]
Please address the request changes.

File: squatwolf-mobile-app/src/assets/icons/LeftArrow.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (FaaiqKhan) [reply squatwolf-mobile-app/src/assets/icons/LeftArrow.tsx:8]: These constants will be moved to the theme file after getting the final designs.

File: squatwolf-mobile-app/src/components/BannerItem.tsx (hunk 4 above)
Comment (FaaiqKhan) [reply squatwolf-mobile-app/src/components/BannerItem.tsx:26]: Picked random color to differentiate the background of the banner this color will be picked from the theme after the final design.

File: squatwolf-mobile-app/src/components/BannerCarousel.tsx (hunk 3 above)
Comment (FaaiqKhan) [reply squatwolf-mobile-app/src/components/BannerCarousel.tsx]: Fixed

**Pull request merged**